Java/E-montazysta/src/main/java/com/emontazysta/model/Manager.java

```diff
@@ -49,5 +49,5 @@ public Manager(Long id, String firstName, String lastName, String email, String
     private List<DemandAdHoc> demandsAdHocs;
 
     @OneToMany(mappedBy = "acceptedBy")
-    private List<ElementEvent> elementEvents;
+    private List<ElementEvent> elementEvents; //TODO: duplicated from Employee?
```

Comment:
In Employee it says who added ElementEvent, here it says which ElementEvents were accepted by this Manager